### PR TITLE
Week 2 scaffolding

### PR DESCRIPTION
**Pull request by esumitra**

### CODE REVIEW

File: core/src/main/scala/org/cscie88c/core/week2/UtilFunctions.scala [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.cscie88c.core.week2
+
+object UtilFunctions {
+
+  // complete the functions below
+  def maximum(a: Int, b: Int): Int = ???
+  def average(a: Int, b: Int): Double = ???
```
Comment (v-tadipatri) [inline core/src/main/scala/org/cscie88c/core/week2/UtilFunctions.scala:7]: for median:
def median(dataset: List[Double]): Double = ???

File: core/src/test/scala/org/cscie88c/core/week2/UtilFunctionsTest.scala [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  
+  "UtilFunctions" when {
+    "maximum" should {
+      "return maximum of two ints when first integer is greater" in {
+        UtilFunctions.maximum(2, 1) should be (2)
+      }
+      // add more unit tests for maximum below
+    }
+
+    // add unit tests for average below
```
Comment (v-tadipatri) [inline core/src/test/scala/org/cscie88c/core/week2/UtilFunctionsTest.scala:15]: update comment to be median

File: core/src/main/scala/org/cscie88c/core/week2/UtilFunctions.scala (hunk 1 above)
Comment (esumitra) [reply core/src/main/scala/org/cscie88c/core/week2/UtilFunctions.scala]: done
